src/tests/ftest/slurm_setup.py

```diff
@@ -114,44 +114,47 @@ def start_munge(self, user):
         Raises:
             SlurmSetupException: if there is a problem starting munge
         """
-        self.log.info("Starting munge")
+        # Get munge status on all nodes
+        self.log.info("Check munge status")
+        if not self._active_systemctl(self.all_nodes, 'munge'):
+            self.log.info("Starting munge")
```

Comment:
To avoid having to indent everything below here, I'd recommend
```suggestion
        self.log.info("Check munge status")
        if self._active_systemctl(self.all_nodes, 'munge'):
            return
        self.log.info("Starting munge")
```